fxgl-entity/src/main/kotlin/com/almasb/fxgl/entity/components/DataComponents.kt

```diff
@@ -39,6 +39,8 @@ abstract class BooleanComponent
     }
 
     override fun toString() = "${javaClass.simpleName.substringBefore("Component")}($value)"
+
+    override fun isComponentInjectionRequired(): Boolean = false
```

Comment:
I think we should not add this to any abstract Component in this file as we don't know whether the subclasses will need injection.

Reply:
Fixed